### ISSUE TITLE
Optional dependency on @blueprintjs/core causes error when used via native browser module

### ISSUE THREAD

**Issue by erictooth** [nomcopter/react-mosaic#138]
### Bug report

- __Package version(s)__: 4.0.0
- __Browser and OS versions__: Chrome 81/MacOS

#### Steps to reproduce

1. Install react-mosaic and import in in app
1. **Build app using a tool that targets native modules for the browser** such as [Snowpack](https://www.snowpack.dev/)
1. Try to load app in the browser

#### Actual behavior

When loading the app, Chrome shows the error: 
> Uncaught TypeError: Failed to resolve module specifier "@blueprintjs/core"

When building the app, Snowpack shows the warning (expected):
> '@blueprintjs/core' is imported by 'node_modules/react-mosaic-component/lib/util/OptionalBlueprint.js', but could not be resolved.

#### Expected behavior

Using react-mosaic in an app distributed via native modules should not cause an error to be thrown due to an optional dependency. 

For now I've added [babel-plugin-transform-remove-imports](https://www.npmjs.com/package/babel-plugin-transform-remove-imports) to my project to remove `import` lines for blueprintjs. Any idea if there’s a better solution?

**Comment by jbaiter** [comment]
The same thing causes any app that depends on `react-mosaic` to fail building with parcel2:

```
$ parcel build src/index.js
🚨 Build failed.
@parcel/packager-js: `require.resolve` calls for excluded assets are only supported with outputFormat: 'commonjs'
/home/user/project/node_modules/react-mosaic-component/lib/util/OptionalBlueprint.js:32:9
  31 |         // https://github.com/nomcopter/react-mosaic/issues/109
> 32 |         require.resolve('@blueprintjs/core');
>    |         ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  33 |         require.resolve('@blueprintjs/icons');
  34 |         (_a = require('@blueprintjs/core'), Classes = _a.Classes, OptionalBlueprint.Icon = _a.Icon);
```

**Comment by nomcopter** [comment]
Fixed in 602b0d5447d8ffd23a0cbf3fca917c5736735ecd
